src/assets/scss/components/SimilarStories.scss

```diff
@@ -0,0 +1,39 @@
+.similar-stories {
+  width: 100%;
+
+  .title {
+    text-align: center;
+    width: 100%;
+  }
+
+  .card {
+    border: 1px solid black;
+    border-radius: 5px;
+    justify-content: space-between;
+    margin-bottom: 10px;
+    min-height: 60px;
+    padding: 5px;
+    width: 100%;
+
+    .subject {
+      margin-block-end: 0px;
+      margin-block-start: 0px;
+    }
+
+    .info {
+      justify-content: space-between;
+      width: 100%;
+
+      .right {

```

Comment:
Our code practice is that we don't go further than 3 levels on SCSS to avoid high specificity.
Too many levels of inheritance on CSS makes its maintenance higher as classes are less re-usable.

What I recommend here is that you try and use one of the spacing variables that we already have to replace this `.right` class, and for the `.subject` and `.info` classes fo something similar, since they only change margin values, so it doesn't make too much sense to create new classes just for this purpose.

This is the spacing scss: https://github.com/EOS-uiux-Solutions/user-story/blob/main/src/assets/scss/layout/spacing.scss

it basically returns a set of classes using this scheme: `.margin-#{$eos-direction}-#{$eos-key} `

so an example would be `.margin-top-m`, `.margin-bottom-xs`, ...

Reply:
Done